11159 Add more information about anonymous access Preview URL; Limit Anon Access to Draft Versions (#11164)
IQSS/dataverse: doc/sphinx-guides/source/user/dataset-management.rst

Preview URL to Review Unpublished Dataset
=========================================

> Creating a Preview URL for a draft version your dataset allows you to share your dataset (for viewing and downloading of files) before it is published to a wide group of individuals who may not have a user account on the Dataverse installation. Anyone you send the Preview URL to will not have to log into the Dataverse installation to view the unpublished dataset. Once a dataset has been published you may continue to use the General Preview URL for future drafts, but the Anonymous Preview URL will no longer be available.

Reviewer: Interesting. I seem to remember that Preview URLs were deleted on publish but apparently that isn't the case anymore, if it ever was!

Author: Maybe unclear. Trying to say that you can make a new ":general" preview url for subsequent drafts. Didn't mean to say that the prior one would continue to work. I'll re-word it.

> Creating a Preview URL for a draft version your dataset allows you to share your dataset (for viewing and downloading of files) before it is published to a wide group of individuals who may not have a user account on the Dataverse installation. Anyone you send the Preview URL to will not have to log into the Dataverse installation to view the unpublished dataset. Once a dataset has been published you may create new General Preview URLs for subsequent draft versions, but the Anonymous Preview URL will no longer be available.

Reviewer 2: ```suggestion
Creating a Preview URL for a draft version of your dataset allows you to share your dataset (for viewing and downloading of files) before it is published to a wide group of individuals who may not have a user account on the Dataverse installation. Anyone you send the Preview URL to will not have to log into the Dataverse installation to view the unpublished dataset. Once a dataset has been published you may create new General Preview URLs for subsequent draft versions, but the Anonymous Preview URL will no longer be available.
```

#. Go to your unpublished dataset
#. Select the “Edit” button
#. Select “Preview URL” in the dropdown menu
#. In the pop-up select “Create General Preview URL” or "Create Anonymous Preview URL". The latter supports anonymous review by removing author names and other potentially identifying information from citations, version history tables, and some metadata fields (as configured by the administrator).
#. Copy the Preview URL which has been created for this dataset and it can now be shared with anyone you wish to have access to view or download files in your unpublished dataset.

To disable a Preview URL and to revoke access, follow the same steps as above until step #3 when you return to the popup, click the “Disable Preview URL” button.

**Note:** Before distributing an anonymized Preview URL it is recommended that you view the dataset as a potential user to verify that the metadata available does not reveal authorship, etc. 

#. Create Anonymous Preview URL for your unpublished dataset via the Preview URL popup from Edit Dataset button
#. Copy the Anonymous Preview URL to your clipboard
#. Log out of Dataverse application 

> #. Open the dataset via the Anonymous Preview URL you plan to distribute to view the draft dataset as a reviewer would see it 

Reviewer 2: ```suggestion
#. Open the dataset using the Anonymous Preview URL you plan to distribute to view it as a reviewer would.
```

#. Copy the Preview URL which has been created for this dataset and it can now be shared with anyone you wish to have access to view or download files in your unpublished dataset.

To disable a Preview URL and to revoke access, follow the same steps as above until step #3 when you return to the popup, click the “Disable Preview URL” button.

> Before distributing an anonymized Preview URL it is recommended that you view the dataset as a potential user to verify that the metadata available does not reveal authorship, etc. You must first log out of Dataverse then open the dataset via the anonymized Preview URL you plan to distribute. By logging out first, you will be able to view the dataset as it would be seen by the anonymized Preview URL user. It may be necessary for you to further edit your dataset's metadata to remove identifying items before you distribute the anonymized Preview URL.

Reviewer: ```suggestion
Before distributing an anonymized Preview URL it is recommended that you view the dataset as a potential user to verify that the metadata available does not reveal authorship, etc. You must first log out of Dataverse and then open the dataset via the anonymized Preview URL you plan to distribute. By logging out first, you will be able to view the dataset as it would be seen by the anonymized Preview URL user. It may be necessary for you to further edit your dataset's metadata to remove identifying items before you distribute the anonymized Preview URL.
```

I'm suggesting the same change below in the bundle but maybe we can discuss it here. I'm simply adding "and" but perhaps the whole paragraph could be made a bit shorter somehow? In the docs it's fine but in the tooltip (thanks for the screenshot!) it looks a bit big.

![404042237-0be43c5d-9662-4daa-8c82-cf82e2d3cf94](https://github.com/user-attachments/assets/4924c952-f0cf-4795-8f7e-8915175aa8bc)

I assume we're using a tooltip because we already have a lot of text in the popup and we're trying not to overwhelm the user.

Author: Right. And that's what Katie suggested and approved

Reviewer 3: I definitely take the point of having a lot of info in the tool tip here too. I just think there should be someplace where a user can access the list of fields that are anonymized. A link to somewhere in the guide would be fine.

Author: As discussed, I removed the tool tip and replaced it with an update to the popup that includes a link to the documentation that includes detailed steps on how to view an anonymous url as a reviewer would see it
